3.0/relationships.md

```diff
@@ -274,7 +274,7 @@ Lazy Loading can cause the problem of `n*1` when you want to fetch relations for
 
 ```javascript
 const user = yield User.with('profile').fetch()
-console.log(users.toJSON())
+console.log(user.toJSON())
```

Comment:
Can u rename the above variable to `users`? since the query will fetch all the users